src/cgutils.cpp

```diff
@@ -111,12 +111,12 @@ AtomicOrdering get_llvm_atomic_order(enum jl_memory_order order)
 static Value *stringConstPtr(
         jl_codegen_params_t &emission_context,
         IRBuilder<> &irbuilder,
-        const std::string &txt)
+        const Twine &txt)
 {
     Module *M = jl_builderModule(irbuilder);
-    StringRef ctxt(txt.c_str(), txt.size() + 1);
-    Constant *Data = ConstantDataArray::get(irbuilder.getContext(), arrayRefFromStringRef(ctxt));
-    GlobalVariable *gv = get_pointer_to_constant(emission_context, Data, "_j_str", *M);
+    SmallVector<char, 128> ctxt;
```

Comment:
Aren't you missing an initialization here?

Reply:
I was under the impression that toStringRef would use the SmallVector as a backing store if the twine didn't have a single string ref representation.

Reply:
I mean that you don't pass `txt` to anywhere.

Reply:
I think I just misread the code ;)